### DISCUSSION TITLE
Documentation is not front facing enough

### DISCUSSION THREAD

**Discussion by Bamowen** [TagStudioDev/TagStudio#443] [General]
Most of the documentation is located in the doc folder, which makes sense in and of itself. However we could use it to populate the wiki tab of the repo to have a clearer entry point for all users.

**Comment by Nginearing** [comment]
Yeah, I agree, I talked to CyanVoxel the other day about the wiki tab, CyanVoxel forgot that the wiki tab was still accessible as it was supposed to be completely replaced by the docs folder. The wiki tab is much more obvious to less techy users. If someone didn't know what the "doc" folder meant then they would have to scroll down quite far in the Read Me to find (a link to the) current documentation.

**Comment by Bamowen** [comment]
In general I think people who might just want to get the software will not even bother looking at the file tree. Readme is a good way to circumvent this, but having a clear point to link to such as wiki (through the tab or the readme) would help I think.

It's also easier to navigate its folder structure than a readme (thinking back on my experience trying to find how to install).
I can try and piece a draft tonight (Germany time)

**Comment by Bamowen** [comment]
So I don't have rights to write in the wiki.
I want to push all the doc folder to the wiki as a start. This way we have a good basis and only Library.md (case sensitive) has to be removed, as a more polished version in library.md exists.

How do we go about this? I think only people with write access to the base repo can clone and push to the wiki (they're 2 separate clone .git and .wiki)

**Reply by Nginearing** [reply]
Hi there, seems most people get round this by forking a repo on GitHub, then cloning only the wiki to their machine by using the wiki.git file, then pushing those changes to the fork, then pushing the changes to the official repo.

The other way is to setup a GitHub Action workflow to automatically sync the doc folder to the wiki Git: see [this](https://github.com/devonfw/devon4j/blob/develop/.github/workflows/github_actions.yml) for reference

**Reply by Nginearing** [reply]
The second one is much more future-proof, I'm going to test it out now on one of my repos.

**Reply by CyanVoxel** [reply]
We switched away from the GitHub wiki in favor of a docs folder to host at a later date due to the writing complications with the wiki.

**Reply by Nginearing** [reply]
> We switched away from the GitHub wiki in favor of a docs folder to host at a later date due to the writing complications with the wiki.

Does this deserve the wontfix label then?

**Reply by Computerdores** [reply]
> We switched away from the GitHub wiki in favor of a docs folder to host at a later date due to the writing complications with the wiki.

maybe at least a link to the index.md?

**Reply by Bamowen** [reply]
I think so @Computerdores. As I see it we need a good easy way to at least link to everything. 
I do understand CyanVoxel's point clearly though.

**Comment by yedpodtrzitko** [comment]
Regarding publishing the docs - it can be done directly on Github quite easily, see https://pages.github.com

**Reply by Bamowen** [reply]
Wouldn't this prevent us from doing it?

> If the first part of the repository doesn’t exactly match your username, it won’t work, so make sure to get it right.

**Reply by CyanVoxel** [reply]
The documentation would be moved to a different repo under the same org in this case

**Reply by Bamowen** [reply]
Would we keep the link to this repo by doing this? I'm not sure how this work, but in general we want to ease and centralize access to doc. If this method would lead to a seamless result for the user, I'm all for it!

**Reply by Nginearing** [reply]
> The documentation would be moved to a different repo under the same org in this case

Also can the wiki be disabled for the meantime as it isn't up-to-date

**Comment by CyanVoxel** [comment]
The documentation is now published and hosted to an MkDocs site via GitHub Pages as of #460, at [docs.tagstud.io](https://docs.tagstud.io).
I've also put the link to this site near the top of the README and in the project sidebar, so I think this discussion can be safely closed now.

**Reply by Toby222** [reply]
(brief bafflement as I read the domain as "Tag Stud" for a moment)
